ddtrace/contrib/flask/middleware.py

```diff
@@ -20,6 +21,11 @@
 SPAN_NAME = "flask.request"
 
 
+@cached()
+def _normalize_resource(resource):
+    return compat.to_unicode(resource).lower()
+
+
 class TraceMiddleware(object):
     @deprecated(message="Use patching instead (see the docs).", version="1.0.0")
```

Comment:
you probably don't need to update this middleware, it is deprecated and is unlikely that it is used by anyone 🤞🏻

Reply:
yeah I wasn't sure but decided to touch it anyway, just in case not everybody has migrated away.